sensor/kubernetes/fake/openshift.go

```diff
@@ -0,0 +1,40 @@
+package fake
+
+import (
+	v1 "github.com/openshift/api/config/v1"
+	fakeAppVersioned "github.com/openshift/client-go/apps/clientset/versioned/fake"
+	fakeConfigVersioned "github.com/openshift/client-go/config/clientset/versioned/fake"
+	fakeOperatorVersioned "github.com/openshift/client-go/operator/clientset/versioned/fake"
+	fakeRouteVersioned "github.com/openshift/client-go/route/clientset/versioned/fake"
```

Comment:
Is this the same import just named differently four times? Or am I missing something here?

Reply:
There are not the same imports, the path changes after `client-go` to `apps`, `config`, `operator`, and `route` respectively 🙂

Reply:
I will just credit this oversight to insufficient caffeine levels from yesterday :roll_eyes: 
Thanks for the clarification